src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -196,6 +214,21 @@ public static TypedData ToRpc(this object value, ILogger logger)
             return typedData;
         }
 
+        private static bool RawBodyBytesRequested(Capabilities capabilities)
+        {
+            return string.Equals(capabilities.GetCapabilityState(LanguageWorkerConstants.RawHttpBodyBytes),
```

Comment:
Only check if the state is not null. A language worker would send capability
```
{"RawHttpBodyBytes", "state can be json in future if needed."}
```
An older version of  python worker would simple not send this capability on init